.gitignore

```diff
@@ -11,3 +11,8 @@ release.properties
 .DS_Store
 .idea
 /package-lock.json
+work/
+
+src/main/webapp/css/**/prism*.css
+src/main/webapp/js/**/prism*.js
+src/main/webapp/js/index.js
```

Comment:
Maybe it makes more sense to use a subfolder for the generated files, otherwise it is hard in IntelliJ to see the difference between copied files and created files.

Reply:
sure